packages/breadcrumb/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@zendeskgarden/container-breadcrumb",
+  "description": "Containers relating to breadcrumb in the Garden Design System",
+  "license": "Apache-2.0",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "homepage": "https://garden.zendesk.com/",
+  "repository": "https://github.com/zendeskgarden/react-containers",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-containers/issues"
+  },
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../utils/scripts/build.sh"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.6.1",
+    "react": "16.8.0-alpha.0",
+    "react-dom": "16.8.0-alpha.0"
+  },
+  "keywords": [
+    "a11y",
+    "accessible",
+    "container",
+    "garden",
+    "headless",
+    "react",
+    "zendesk"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "zendeskgarden:library": "GardenBreadcrumb",
```

Comment:
Can you remind me why we need these `"zendeskgarden:..."` values?

Reply:
These inform the Webpack build of the global library name to attach to `window` (silly UMD builds)

Reply:
hmm so I should probably change this GardenBreadcrumbContainer so as not to clash with react-components if consumed via umd builds?

Reply:
Have we looked into not including UMD builds with this? If no one is consuming it, might be added complexity we could live without.

Reply:
yeah perhaps we should just ship cjs

Reply:
So I might create another story just to address this and remove umd builds from all packages in a separate PR.